content/guides/composite_monitors.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Guide to Composite Monitors
+kind: guide
+listorder: 9
+---
+
+***If you're unfamiliar with the basics of Datadog Monitors, first read the [Guide to Monitors](/guides/monitors)***
+
+---
+
+Composite monitors let you combine many individual monitors into one so that you can define more specific alert conditions. Choose two or more existing monitors - monitor A and monitor B, say - and then set a trigger condition using boolean operators (e.g. “A && B”). The composite monitor will trigger when its individual monitors' statuses simultaneously have values that cause the composite's trigger condition to be true.
+
+After you create a composite monitor, you can modify it just as you would an individual monitor. To change the trigger condidtion, edit the composite monitor. To change one of its constituent monitors, though - e.g. to disable its notifications in favor of the composite notification - simply edit that individual monitor.
+
+## Creating composite monitors
+
+In Datadog, go to the New Monitor page and select 'Composite Monitor' from the list of monitor types:
+
+![choose-composite-type](/static/images/composite_monitors/select-monitor-type.png)
+
+### Choose individual monitors
+
+You can choose up to 10 individual monitors to use in the new composite monitor. When choosing monitors, the following rules apply:
+
+* No individual monitor may itself be a composite monitor
+* Individual monitors may be all simple alert monitors, all multi-alert monitors, or a combination of simple and multi-alert
+* If more than one multi-alert monitor is chosen, then **all** multi-alert monitors in the composite monitor must:
+  1. Use the same grouping (e.g. a monitor grouped by `{host}` cannot be used with one grouped by `{environment}`) AND the same group ordering (e.g. a monitor grouped by `{host,device}` cannot be used with one grouped by `{device,host}`)
```

Comment:
this is not exactly true
e.g. a multi-alert with a average_by: host result for host:i-3892738 and another multi-alert with a average_by:device_host and a result for device_host:i-3892738 --> these monitor results will undergo whatever operation is in the query and be evaluated

so even though the tag (host:i-3892738 !== device_host:i-3892738) does not equal each other the result will be evaluated because the group VALUE i-3892738 == i-3892738 is the same

of course availability-zone and host will likely never lead to the same reporting source, so it LOOKS like the same group_by keys must be necessary but it is only an indirect consequence of those group-bys never being able to produce the same reporting sources.